compiler/testData/codegen/box/functions/kt50986.kt

```diff
@@ -0,0 +1,5 @@
+// TARGET_BACKEND: JVM_IR
```

Comment:
Please, add `// CHECK_BYTECODE_LISTING` directive and re-run the test.
It'll produce `kt50986.txt` file containing generated bytecode signatures.
Check that it passes in the old back-end as well (to make sure that we don't break ABI).